### DISCUSSION TITLE
Laser GRBL Keeps disconnecting while engraving

### DISCUSSION THREAD

**Discussion by Mandylane** [arkypita/LaserGRBL#2115] [Q&A]
Hello,

I recently bought a TwoTrees laser engraver and i am using laserGRBL to engrave

I build one mini simple engraver. I've been facing many issues, but this one I faced, I've got no idea.
The laser engraver stops at the middle of the run. The laserGRBL makes a beep sounds (pc) then it just stops running and not running any commands. Sometimes it stops and leaves a small hole burn and I can't continue with the job. I restart the software But this happens again at every run.
It doesn't even show little window saying 'Problem detected'. Please any advice would be great. PC makes a beep sound and engraving stops, it almost look like it freeze.

Here is more detail about my printer.
I use TwoTrees TTS-55

**Comment by arkypita** [comment]
Could you please open your session log (menu ?, open session log) and paste it here?
